package.json

```
@@ -13,6 +13,9 @@
   "license": "CC BY-SA 4.0",
   "author": "Open Terms Archive",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "lint:markdown": "markdownlint *.md content/**/*.md"
```

Sorry, I was not clear. It is very useful to have the `test` script, as it is one of the “lifecycle” scripts of NPM, which means it has special meaning (for example, if you write `npm test` it would work, you don't need to write `npm run test`). By “replacing”, I meant replacing the contents of the script 🙂 

```suggestion
    "test": "markdownlint *.md content/**/*.md"
```

You could also do the following if you want to keep an explicit name for subtasks:

```suggestion
    "lint:markdown": "markdownlint *.md content/**/*.md",
    "test": "npm run lint:markdown"
```

---

Thanks for the clarification, I did a fixup.